frontend/src/components/Footer.js

```diff
@@ -57,82 +57,123 @@ const SocialButton = ({ children, label, href }) => {
-            </SocialButton>
-            <SocialButton
-              label={"Twitter"}
-              href={"https://twitter.com/moonstreamto"}
-            >
-              <FaTwitter />
-            </SocialButton>
-            <SocialButton
-              label={"Github"}
-              href={"https://github.com/bugout-dev/moonstream"}
-            >
-              <FaGithub />
-            </SocialButton>
-            <SocialButton
-              label={"LinkedIn"}
-              href={"https://www.linkedin.com/company/moonstream/"}
-            >
-              <FaLinkedin />
-            </SocialButton>
-          </Stack>
-        </Stack>
-        {Object.values(SITEMAP).map((category, colIndex) => {
-          return (
-            <Stack align={"flex-start"} key={`footer-list-column-${colIndex}`}>
+const Footer = () => {
+  const ui = useContext(UIContext);
+  return (
+    <Box
+      bg={BACKGROUND_COLOR}
+      textColor="white"
+      borderTop="1px"
+      borderColor="white"
+      px="7%"
+    >
+      <Container as={Stack} maxW={"8xl"} py={10}>
+        <Flex direction={["column", "column", "row"]}>
+          <Stack spacing={6}>
+            <Box pb={ui.isMobileView ? "40px" : "0px"}>
+              <Link href="/" alignSelf="center">
+                <ChakraImage
+                  alignSelf="center"
+                  w="160px"
+                  src={PRIMARY_MOON_LOGO_URL}
+                  alt="Go to app root"
+                />
+              </Link>
+            </Box>
+            {!ui.isMobileView && (
               <>
-                <ListHeader>{category.title}</ListHeader>
-                {category.children.map((linkItem, linkItemIndex) => {
-                  return (
-                    <RouterLink
-                      passHref
-                      href={linkItem.path}
-                      key={`footer-list-link-item-${linkItemIndex}-col-${colIndex}`}
-                    >
-                      <Link {...LINKS_SIZES}>{linkItem.title}</Link>
-                    </RouterLink>
-                  );
-                })}
+                <Flex justifyContent="start">
+                  <Link href="//privacy-policy">Privacy policy</Link>
+                  <Link href="//tos" ml="20px">
+                    Terms of Service
+                  </Link>
+                </Flex>
+                <Text fontSize={"sm"}>
+                  © {moment().year()} Moonstream.to All rights reserved
+                </Text>
               </>
-            </Stack>
-          );
-        })}
-      </SimpleGrid>
-    </Container>
-  </Box>
-);
+            )}
+          </Stack>
+          <Spacer />
+          <Flex direction="column" pb={ui.isMobileView ? "40px" : "0px"}>
+            <Text fontWeight="semibold" mb="20px">
+              Follow Us
+            </Text>
+            <Flex width="158px" justifyContent="space-between">
+              <SocialButton label={"Discord"} href={"/discordleed"}>
```

Comment:
While we're here it would be good to have mixpanel reports on the social buttons.